bedrock/firefox/tests/test_views.py

```diff
@@ -678,3 +676,60 @@ def test_firefox_welcome_page1(self, render_mock):
         req.locale = "en-US"
         views.firefox_welcome_page1(req)
         render_mock.assert_called_once_with(req, "firefox/welcome/page1.html", ANY, ftl_files="firefox/welcome/page1")
+
+
+class TestSMSSendToDevice(TestCase):
+    def setUp(self):
+        self.url = "/en-US" + reverse("firefox.sms-send-to-device-post")
+        patcher = patch("bedrock.firefox.views.TWILIO_CLIENT")
+        self.mock_create = patcher.start()
+        self.addCleanup(patcher.stop)
+
+    def _do_post(self, params):
+        return self.client.post(
+            self.url,
+            data=params,
+            follow=True,
+            HTTP_X_REQUESTED_WITH="XMLHttpRequest",
```

Comment:
I don't think this is required. I don't see a use of `request.is_ajax()` anywhere. Not going to hurt anything though.